uaa/src/main/webapp/WEB-INF/spring-servlet.xml

```diff
@@ -248,7 +248,13 @@
     </bean>
 
     <bean id="contentSecurityPolicyFilter"
-          class="org.cloudfoundry.identity.uaa.security.web.ContentSecurityPolicyFilter"/>
+          class="org.cloudfoundry.identity.uaa.security.web.ContentSecurityPolicyFilter">
+        <property name="cspAllowedScriptSrc"
+                  value="#{@config['contentSecurityPolicy']==null ? T(java.util.Arrays).asList('self') :
+                           @config['contentSecurityPolicy']['allowed']==null ? T(java.util.Arrays).asList('self') :
+                           @config['contentSecurityPolicy']['allowed']['scriptSrc']==null ? T(java.util.Arrays).asList('self') :
```

Comment:
I recommend to simply use csp and omit allowed e.g. uaa.yml from 

uaa.csp.script-src:
  - 'self'
  - 'unsafe-inline'

would place csp section in uaa-release
https://github.com/cloudfoundry/uaa-release/blob/develop/jobs/uaa/templates/config/uaa.yml.erb#L845

csp defines different section .e.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy so currently we need only script-src , later we could add other in this section